tests/test_http_auth.py

```diff
@@ -38,7 +38,7 @@ def _scram_auth(self, user, password):
             _, headers, status = self.http_con_request(con, {}, path="token")
             self.assertEqual(status, 401)
             self.assertEqual(
-                headers, headers | {"www-authenticate": "scram-sha-256"}
+                headers, headers | {"www-authenticate": "SCRAM-SHA-256"}
```

Comment:
Here is the one place where we were depending on the lowercasing of the header values in the testbase class.